manageiq-schema.gemspec

```diff
@@ -25,10 +25,10 @@ Gem::Specification.new do |spec|
   spec.add_dependency "more_core_extensions",    ">= 3.5", "< 5"
   spec.add_dependency "pg"
   spec.add_dependency "pg-pglogical",            "~> 2.1.1"
-  spec.add_dependency "rails",                   "~>6.0.0"
+  spec.add_dependency "rails",                   "~> 6.0.0", "> 6.0.6.1"
 
-  spec.add_development_dependency "manageiq-style", ">= 1.5.2"
+  spec.add_development_dependency "manageiq-style", ">= 1.5.3"
```

Comment:
To keep it consistent, can you also make this change on master?

Reply:
https://github.com/ManageIQ/manageiq-schema/pull/778